### PR TITLE
add flux benchmark

### PR DESCRIPTION
**Pull request by CarlosGomes98**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -208,8 +208,9 @@ MLCommons project work is tracked with issue trackers and pull requests. Modify
  | resnet | 67840.00 | 85.86 | 0.64 | 0.007 | 5 |
  | ssd | 256.00 | 4.00 | 0.00| 0.000 | 5 |
  | ssd | 4096.00 | 15.13 | 0.50 | 0.033 | 5 |
- | stable_diffusion | 512.00 | 5071.43 | 257.54 | 0.051 | 10 |
- | stable_diffusion | 2048.00 | 1961.54 | 165.43 | 0.084 | 10 |
+ | flux | 1024.00 | 8768716.8 | 592157.22 | 0.068 | 10 |
```
Comment (ShriyaRishab) [inline CONTRIBUTING.md]: Can we rename to flux.1 everywhere?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -208,8 +208,9 @@ MLCommons project work is tracked with issue trackers and pull requests. Modify
  | resnet | 67840.00 | 85.86 | 0.64 | 0.007 | 5 |
  | ssd | 256.00 | 4.00 | 0.00| 0.000 | 5 |
  | ssd | 4096.00 | 15.13 | 0.50 | 0.033 | 5 |
- | stable_diffusion | 512.00 | 5071.43 | 257.54 | 0.051 | 10 |
```
Comment (ShriyaRishab) [inline CONTRIBUTING.md:211]: Let's keep stable_diffusion even though its retired since this table helps decide N for future benchmarks and having as much data here as possible is good.

Comment (pgmpablo157321) [reply CONTRIBUTING.md:211]: Agree with keeping stable diffusion. But I also see there are some other benchmarks missing from this table like the Llama models. Should we open an issue to add those to this table?

**Review by pgmpablo157321 — COMMENTED** [review]
Once the model is renamed to flux1, we can merge this from my side

File: CONTRIBUTING.md (hunk 2 above)
Comment (ShriyaRishab) [reply CONTRIBUTING.md:211]: This table is not required to be filled in but is more for guidance. So filling this in for a benchmark can help future benchmarks because they can refer to this table and make similar decisions. 

In the llama3.1 405b case, we actually didn't follow the recommendations and it is actually an outlier so I don't think we should add those data points here.

File: training_rules.adoc [hunk 3]
```diff
@@ -533,14 +541,12 @@ The SWG must come to majority consensus to approve a submission that fails the R
...
-* Stable Diffusion
+* Flux
 ** 10 runs per submission
-** Checkpoint must be collected every 512,000 images. CEIL(512000 / global_batch_size) if 512000 is not divisible by GBS.
-** The collected checkpoints may be evaluated freely (in order, out of order, some checkpoints may be skipped), provided that:
-1. FID and CLIP scores must to be submitted for all collected checkpoints (up to the first checkpoint with a passing score) for 1/10 of the runs.
-2. FID and CLIP scores must to be submitted for the last two checkpoints (the first checkpoint with a passing score and the one before it) for 9/10 of the runs.
-** evaluation is done offline, the time is not counted towards the submission time.
-** A passing score is FID<=90 and CLIP>=0.15
+** Checkpoint must be collected every 262,144 images. CEIL(262144 / global_batch_size) if 262144 is not divisible by GBS.
```
Comment (ShriyaRishab) [inline training_rules.adoc]: Why should checkpoint be collected for FLUX.1? We only care about eval loss right?

File: training_rules.adoc [hunk 4]
```diff
@@ -319,6 +319,14 @@ The MLPerf verifier scripts checks all hyperparameters except those with names m
  |llama2_70b_lora |adamw |opt_learning_rate_warmup_ratio | unconstrained |ratio of steps out of training for linear warmup during initial checkpoint generation. This only affects the learning rate curve in the benchmarking region. |See PR (From Habana, TODO Link) |v5.1
  |llama2_70b_lora |adamw |opt_learning_rate_training_steps | unconstrained |Step when the end of cosine learning rate curve is reached. Learning rate cosine decay is in range (opt_learning_rate_warmup_steps + 1,opt_learning_rate_decay_steps]. |See PR (From Habana, TODO Link) |v5.1
  |llama2_70b_lora |adamw |opt_base_learning_rate |unconstrained | base leraning rate |See PR (From Habana, TODO Link) |v5.1
+ |flux |adamw |global_batch_size |unconstrained |The global batch size for training |link: https://github.com/pytorch/torchtitan/blob/3d7ea4b6b7e5cd2decf11a1c37581d1cd3678fe0/torchtitan/config_manager.py#L198[reference_code] |v5.1
```
Comment (ShriyaRishab) [inline training_rules.adoc]: can we rename to flux.1 for all hyper parameters?

File: training_rules.adoc (hunk 3 above)
Comment (CarlosGomes98) [reply training_rules.adoc]: thanks for catching this, indeed doesnt make sense, it should say evaluation

File: training_rules.adoc [hunk 5]
```diff
@@ -533,14 +541,12 @@ The SWG must come to majority consensus to approve a submission that fails the R
...
-** Checkpoint must be collected every 512,000 images. CEIL(512000 / global_batch_size) if 512000 is not divisible by GBS.
-** The collected checkpoints may be evaluated freely (in order, out of order, some checkpoints may be skipped), provided that:
-1. FID and CLIP scores must to be submitted for all collected checkpoints (up to the first checkpoint with a passing score) for 1/10 of the runs.
-2. FID and CLIP scores must to be submitted for the last two checkpoints (the first checkpoint with a passing score and the one before it) for 9/10 of the runs.
-** evaluation is done offline, the time is not counted towards the submission time.
-** A passing score is FID<=90 and CLIP>=0.15
+** Evaluation must be run every 262,144 samples. CEIL(262144 / global_batch_size) if 262144 is not divisible by GBS.
+** For the evaluation algorithm:
+*** Timestep values must be {0, 1/8, 2/8, 3/8, 4/8, 5/8, 6/8, 7/8}
+*** Each timestep value must be used an equal number of times
```
Comment (ShriyaRishab) [inline training_rules.adoc]: We don't need this evaluation section right? Everyone just needs to use the eval dataset along with its timestamps which should already ensure that everything is used equal number of times

**Comment by ShriyaRishab** [comment]
Please update these tables - 

https://github.com/mlcommons/training_policies/blob/master/training_rules.adoc#41-closed-division

https://github.com/mlcommons/training_policies/blob/master/training_rules.adoc#94-quality-measure

https://github.com/mlcommons/training_policies/blob/master/training_rules.adoc#12-benchmark-results

File: training_rules.adoc (hunk 5 above)
Comment (CarlosGomes98) [reply training_rules.adoc]: Youre right, will update

**Review by ShriyaRishab — APPROVED** [review]
